### PR TITLE
initial draft of documentation.

### PR DESCRIPTION
**Pull request by chandraprakash**

### CODE REVIEW

File: src/connection_manager.rs [hunk 1]
```diff
@@ -101,10 +111,26 @@ impl ConnectionManager {
         Ok(vec![Endpoint::Tcp(local_addr)])
     }
 
+    /// This method tries to connect (bootstrap to exisiting network) to the default or provided
+    /// list of bootstrap nodes. If the bootstrap list is `Some`, the method will try to connect to
+    /// all of the endpoints specified in `bootstrap_list`. It will return once connection with any of the
```
Comment (dirvine) [inline src/connection_manager.rs]: is this correct -> all of the endpoints in the bootstrap list ?

Comment (chandraprakash) [inline src/connection_manager.rs]: I have not mentioned sequentially as it may change.

Comment (chandraprakash) [inline src/connection_manager.rs]: It will try the list until it gets one successful connection.

Comment (dirvine) [inline src/connection_manager.rs]: OK merging now, sounds reasonable, thx for this

**Pull request merged**